lisa/base_tools/wget.py

```diff
@@ -90,3 +93,45 @@ def verify_internet_access(self) -> bool:
                 f"Internet is not accessible, exception occurred with wget {e}"
             )
         return False
+
+    @classmethod
+    def _windows_tool(cls) -> Optional[Type[Tool]]:
+        return WindowsWget
+
+
+class WindowsWget(Wget):
+    @property
+    def command(self) -> str:
```

Comment:
It doesn't need to overwrite this to empty. Add `_check_exists` and always return True.